src/bun.js/test/jest.zig

```diff
@@ -1080,6 +1082,33 @@ pub const Expect = struct {
         return .zero;
     }
 
+    pub fn toMatchSnapshot(this: *Expect, globalObject: *JSC.JSGlobalObject, callframe: *JSC.CallFrame) callconv(.C) JSValue {
+        defer this.postMatch(globalObject);
+
+        const _arguments = callframe.arguments(1);
```

Comment:
We can do `callframe.argumentCount()` here 

That being said, it shouldn't be an error I don't think. It should just ignore extra arguments, unless Jest does this too